spring-cloud-kubernetes-fabric8-discovery/src/test/java/org/springframework/cloud/kubernetes/fabric8/discovery/ServicePortSecureResolverTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013-2019 the original author or authors.
```

Comment:
move to junit-5 in here (there will be more like these as I slowly go through this package)